CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# v0.15.0
+
+- `pairplot()`, `conditional_pairplot()`, and `conditional_corrcoeff()` should now 
+be imported from `sbi.analysis` instead of `sbi.utils` (#394).
+- Active subspaces for sensitivity analysis (#394)
```

Comment:
Maybe link to the tutorial for sensitivity analysis and make this the first point of the changelog (since it's more interesting?)